--- origsrc/gcr-3.4.1/gck/tests/Makefile.am	2011-11-21 03:10:59.000000000 -0600
+++ src/gcr-3.4.1/gck/tests/Makefile.am	2012-05-04 01:04:53.912628300 -0500
@@ -48,10 +48,10 @@ check-memory: perform-memcheck
 
 all-local: $(check_PROGRAMS)
 
-lib_LTLIBRARIES = libmock-test-module.la
+noinst_LTLIBRARIES = libmock-test-module.la
 
 libmock_test_module_la_LDFLAGS = \
-	-avoid-version
+	-avoid-version -no-undefined -rpath `pwd`
 
 libmock_test_module_la_CFLAGS = \
 	-I$(top_srcdir)/gck \
--- origsrc/gcr-3.4.1/gcr/tests/Makefile.am	2012-02-10 09:59:58.000000000 -0600
+++ src/gcr-3.4.1/gcr/tests/Makefile.am	2012-05-04 01:36:39.755636200 -0500
@@ -14,7 +14,7 @@ INCLUDES = \
 LDADD = \
 	$(top_builddir)/gcr/libgcr-testable.la \
 	$(top_builddir)/egg/libegg.la \
-	$(top_builddir)/gck/libgck-testable.la \
+	$(top_builddir)/gck/tests/libmock-test-module.la \
 	$(GTK_LIBS) \
 	$(GLIB_LIBS) \
 	$(LIBGCRYPT_LIBS)
